pamphlet/agenda.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "biocommons hackathon"
+subtitle: "San Francisco, CA"
+date: "September 2023"
+page-background: "img/gg-bridge.jpg"
+page-background-opacity: 0.1
+---
+
+### Sunday, September 17th
+
+| Time | Event |
+| -- | ------------------- |
+| 8:45 | Arrival &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; |
+| 9:00 | Welcome and ice breaker |
+| 9:45 | Project presentations |
+| 10:15 | Form teams |
+| 10:30 | Hack! |
+| 12:00 | Lunch break |
+| 13:00 | Hack! |
+| 15:00	| Afternoon break |
+| 15:15 | Hack! |
+| 16:45 | Wrap up and early success stories |
+| 17:00 | Triage issues |
+| 18:00 | Transit |
+| 18:30 | Dinner at Chateau de Reece |
```

Comment:
Include [Truly Mediterranean](https://www.trulyeats.com/catering-menu) wraps, platters, and kababs